ExilenceNextApp/src/services/pricing.service.ts

```diff
@@ -50,7 +51,7 @@ function priceItem(item: IPricedItem, prices: IExternalPrice[]) {
         price = prices.find(
           p =>
             item.name.startsWith(p.name) &&
-            p.links === item.links &&
+            (item.links < 5 || p.links === item.links) &&
```

Comment:
Unsure about this, are we sure about this? Ninja makes a difference in the pricing of unique items for 5 or 6 links 
![image](https://user-images.githubusercontent.com/5909669/70712626-7b998000-1ce4-11ea-861e-4b4106b6069f.png)

Reply:
This should be correct? We basically group them as 0-4, 5 or 6 now. We only compare the links if they are 5 or 6.

Reply:
That's right! my wrong